src/main/java/com/hotels/plunger/TapUtil.java

```diff
@@ -48,4 +55,22 @@
     return null;
   }
 
+  /** Create a new {@link JobConf} which has all the values in the given {@link Configuration} */
+  static JobConf newJobConf(Configuration conf) {
```

Comment:
Actually, I didn't want to create another Util class to add this 2 little methods.